packages/devtools_app/lib/src/screens/memory/panes/diff/widgets/classes_table_diff.dart

```diff
@@ -199,15 +199,23 @@ class ClassesTableDiff extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return FlatTable<DiffClassStats>(
-      columns: _columns,
-      columnGroups: _columnGroups,
-      data: classes.classes,
-      dataKey: 'ClassesTableDiff',
-      keyFactory: (e) => Key(e.heapClass.fullName),
-      selectionNotifier: selection,
-      defaultSortColumn: _retainedSizeDeltaColumn,
-      defaultSortDirection: SortDirection.descending,
+    return ValueListenableBuilder(
+      valueListenable: classes.filter,
+      builder: (_, __, ___) {
+        final data = classes.filtered();
+        final dataKey = 'ClassesTableSingle-$data';
```

Comment:
what is an example of what $data yields? if it returns something like `InstanceOf List<DiffClassStats`, then this won't be unique